src/app/(admin)/new_dashboard/guides/components/Editor/index.tsx

```diff
@@ -1,30 +1,26 @@
 import dynamic from "next/dynamic";
-import { useState } from "react";
+import { ChangeEvent, useState } from "react";
 import "react-quill/dist/quill.snow.css";
 import { Box, Button, TextField } from "@mui/material";
 import SaveAsIcon from "@mui/icons-material/SaveAs";
-import { UseFormRegister, UseFormSetValue } from "react-hook-form";
-import { CreateGuideRequest } from "@/api/types/Guide";
 
 const DynamicReactQuill = dynamic(() => import("react-quill"));
 
 interface Props {
   initialTitle?: string;
+  initialIssue?: string;
```

Comment:
what is initial issue for? now the issue will be changed to an array of categories, better to delete all the issue logic from here, when we merge the rest of the pr's all of this logic will be redundent, and we have written existing functions to handle the issue adding.